### ISSUE TITLE
Position of the 'toggle sidebar' button

### ISSUE THREAD

**Issue by agreensh** [macosui/macos_ui#507]
## Use case

Match the native positioning of the 'toggle sidebar' button

## Proposal

Have the button for toggling the sidebar in the same position as native macos apps.
When toggling, in native apps, the position stays the same.

[Current Flutter macos_ui version]
<img width="262" alt="flutter" src="https://github.com/macosui/macos_ui/assets/1016469/0301ad2d-4fc0-4edc-bc21-82c006a71250">

[Native version]
<img width="170" alt="native" src="https://github.com/macosui/macos_ui/assets/1016469/00483f54-83bb-46df-9f14-61d0399d2c6f">

**Comment by Adrian-Samoticha** [comment]
You’re right. It appears that the native ‘toggle sidebar’ button remains stationary when the sidebar is clicked. I’ll try to investigate this issue.
